cmd/tools/grafana/grafana.go

```diff
@@ -671,7 +690,8 @@ func init() {
 	Cmd.PersistentFlags().StringVarP(&opts.addr, "addr", "a", "http://127.0.0.1:3000", "address of Grafana server (IP, FQDN or hostname)")
 	Cmd.PersistentFlags().StringVarP(&opts.token, "token", "t", "", "API token issued by Grafana server for authentication")
 	Cmd.PersistentFlags().StringVarP(&opts.dir, "directory", "d", "grafana/dashboards/", "when importing, directory that contains dashboards.\nWhen exporting, directory to write dashboards to")
-	Cmd.PersistentFlags().StringVarP(&opts.folder, "folder", "f", grafanaFolderTitle, "Grafana folder name for the dashboards")
+	Cmd.PersistentFlags().StringVarP(&opts.folder[0].folderName, "harvestfolder", "H", grafanaFolderTitle, "Grafana folder name for the C mode dashboards")
+	Cmd.PersistentFlags().StringVarP(&opts.folder[1].folderName, "7modefolder", "f", grafana7modeFolderTitle, "Grafana folder name for the 7 mode dashboards")
```

Comment:
maybe `--folder-7mode` and no short option?

Reply:
done